Makefile

```
@@ -36,36 +34,27 @@ fontpack_binary: arm_tools fontpack_library
+	$(MAKE) -C RefPrj/RTE/Release -f Makefile clean
+	rm -f Release/OSTC4_Firmware*.bin Release/OSTC4_FontPack*.bin Release/OSTC4_RTE*.bin
+
+distclean: clean packer_clean
+
+test:
+	@echo "No tests defined"
+
+include arm_build_tools.mk
 
-.PHONY: firmware fontpack rte fontpack_library firmware_binary fontpack_binary rte_binary all clean print_version arm_tools arm_tools_version arm_tools_clean packer packer_clean distclean
+.PHONY: firmware fontpack rte fontpack_library firmware_binary fontpack_binary rte_binary all clean print_version packer packer_clean distcleantest
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)